spec/system/provider_interface/authentication_fallback_spec.rb

```diff
@@ -68,7 +68,7 @@ def then_i_do_not_receive_an_email_with_a_signin_link
 
   def then_i_receive_an_email_with_a_signin_link
     open_email(@email)
-    expect(current_email.subject).to have_content t('authentication.sign_in.email.subject')
+    expect(current_email.subject).to have_content 'Sign in - manage teacher training'
```

Comment:
Any reason it's not the full `Sign in  - manage teacher training applications` ?

Reply:
Nop :)